packages/devextreme/testing/testcafe/helpers/accessibility/accessibilityUtils.ts

```diff
@@ -2,16 +2,18 @@
 import { axeCheck, createReport } from '@testcafe-community/axe';
 import { ElementContext, RunOptions } from 'axe-core';
 
+export { ElementContext };
```

Comment:
let's remove it and use only import from `axe-core`?

Reply:
Fixed. Thanks for your review)